pkg/orchestrator/gitHubActions.go

```diff
@@ -26,7 +26,6 @@ type GitHubActionsConfigProvider struct {
 	runData     run
 	jobs        []job
 	jobsFetched bool
-	currentJob  job
```

Comment:
Unit tests should be adjust as there are some usages of `currectJob` field and `guessCurrentJob` method